modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Panel/PanelBody.tsx

```diff
@@ -55,15 +54,6 @@ export function PanelSimpleBody({
 	return (
 		<div className="object-admin-panel__simple-body">
 			<div className="object-admin-panel__simple-body__content-left">
```

Comment:
Hey @diogomelo09, since we don't have the drag icon anymore, we have to adjust the style, so you can add a `margin-left: 1rem;` for this class, in `Panel.scss`.